<map id="dpp::task_cancelled_exception" name="dpp::task_cancelled_exception">
<area shape="rect" id="node1" title="Represents the cancellation of a task. Will be thrown to the awaiter of a cancelled task." alt="" coords="113,261,263,303"/>
<area shape="rect" id="node2" href="$classdpp_1_1exception.html" title="The dpp::exception class derives from std::exception and supports some other ways of passing in error..." alt="" coords="129,185,247,212"/>
<area shape="rect" id="node3" title=" " alt="" coords="5,96,120,123"/>
<area shape="rect" id="node4" title=" " alt="" coords="145,96,231,123"/>
<area shape="rect" id="node5" title=" " alt="" coords="119,5,257,47"/>
<area shape="rect" id="node6" title=" " alt="" coords="256,96,419,123"/>
</map>
